docs/management_api.md

```diff
@@ -73,12 +73,12 @@ curl -v -X POST "http://localhost:8081/models?url=https%3A%2F%2Fs3.amazonaws.com
 
 ### Scale workers
 
-* PUT /models/{model_name}
-** min_worker - optional minimum number of worker processes. MMS will trying maintain min_worker of workers for specified model, default 1.
-** max_worker - optional maximum number of worker processes. MMS will make no more than max_worker of workers for specified model, default to min_worker.
-** number_gpu - optional number of GPU worker processes to create, default 0. If number_pgu exceed, rest of workers will be running on CPU.
-** synchronous - Decides whether the call is synchronous or not, default: false.
-** timeout - Waiting up to the specified wait time if necessary for a worker to complete all pending requests. Use 0 to terminate backend worker process immediately. Use -1 for wait infinitely.", default -1. **Note**, not implemented yet.
+`PUT /models/{model_name}`
+* min_worker - optional minimum number of worker processes. MMS will trying maintain min_worker of workers for specified model, default 1.
```

Comment:
```suggestion
* min_worker - (optional) the minimum number of worker processes. MMS will try to maintain this minimum for specified model. The default value is `1`.
```
I would prefer `min_workers`.

Reply:
How do initial worker and min_worker interact?

Reply:
both min_worker and max_worker will be set to initial_worker during registration. User can change it using scale API.